examples/gopts.c

```diff
@@ -110,38 +111,25 @@ static void removeSpec (void)
 // Main example
 // -----------------
 
-int main (void)
+int main (int argc, const char * const * argv)
 {
+	// normally, the spec should already be mounted
```

Comment:
Because otherwise you can't just run the example. AFAIK there is even a build job that runs all the examples. To make that work, you'd have to mount the spec before running the example and unmount it afterwards. That sounds like CMake nightmare to me.

Reply:
No, it makes sense to have this included in the example (like it is). I only find the text confusing. Somewhere we should tell the user how a real application should do it.

Reply:
Yes, we should probably update `doc/tutorials/command-line-options.md` and the `gopts` plugin README to explain the whole spec mounting a bit more.

Reply:
Yes, that would be amazing.